plugins/scaffolder/src/components/TaskPage/TaskPage.tsx

```diff
@@ -234,29 +240,40 @@ const hasLinks = ({ links = [] }: ScaffolderTaskOutput): boolean =>
  * @public
  */
 export type TaskPageProps = {
+  cancelTaskButtonText?: string;
+  cancellingTaskStatusText?: string;
   loadingText?: string;
 };
 
 /**
  * TaskPage for showing the status of the taskId provided as a param
+ * @param cancelTaskButtonText - Optional text shown for 'Cancel' task button.
+ * @param cancellingTaskStatusText - Optional text shown for the user during cancelling the task.
  * @param loadingText - Optional loading text shown before a task begins executing.
  *
  * @public
  */
-export const TaskPage = ({ loadingText }: TaskPageProps) => {
+export const TaskPage = ({
+  cancelTaskButtonText = 'Cancel',
+  cancellingTaskStatusText = 'Cancelling...',
+  loadingText,
+}: TaskPageProps) => {
```

Comment:
Would you mind making this just `props: TaskPageProps` and unpacking inside the function body instead? Makes the API report much cleaner.